scripts/autogen-version

```diff
@@ -81,7 +81,7 @@ dirty=$(echo "${git_version}" | awk -F - '{print $4}')
 test -n "${commit}" -a -n "${dirty}" && commit="${commit}.${dirty}"
 
 case "${branch}" in
-    master)
+    main|master)
```

Comment:
We should be able to get rid of `master` here altogether, right?
```suggestion
    main)
```

Reply:
I left it so that it would continue to work with existing checkouts of master. I believe that if somebody just does `git pull` on that, it'll start to pull from main, but locally keep looking like its still `master` (this is just guessing though).

Reply:
Hmm, existing checkouts of `master` should have a version of this script which just looks for `master`, so I think they should be fine if we remove `master`, unless they plan to copy this file out-of-tree.

Reply:
GitHub will redirect master pulls to main, so I believe what will happen is that people will stay on the master branch locally, but still get the new files from main (unless they explicitly switch to main of course).

Reply:
Makes sense.